app/uk/gov/hmrc/rasapi/connectors/DesConnector.scala

```diff
@@ -113,31 +113,31 @@ class DesConnector @Inject()(
 
     result.map(response => resolveResponse(response, userId, member.nino, apiVersion)).recover {
       case badRequestEx: BadRequestException =>
-        Logger.error(s"[DesConnector] [getResidencyStatus] Bad Request returned from des. The details sent were not " +
+        Logger.error(s"[DesConnector][getResidencyStatus] Bad Request returned from des. The details sent were not " +
           s"valid. userId ($userId).")
         Right(ResidencyStatusFailure(error_DoNotReProcess, "Internal server error."))
       case notFoundEx: NotFoundException =>
         Right(ResidencyStatusFailure(error_MatchingFailed, "Cannot provide a residency status for this pension scheme member."))
       case Upstream4xxResponse(_, 429, _, _) =>
-          Logger.error(s"[DesConnector] [getResidencyStatus] Request could not be sent 429 (Too Many Requests) was sent " +
+          Logger.error(s"[DesConnector][getResidencyStatus] Request could not be sent 429 (Too Many Requests) was sent " +
             s"from the HoD. userId ($userId).")
           Right(ResidencyStatusFailure(error_TooManyRequests, "Too Many Requests."))
       case requestTimeOutEx: RequestTimeoutException =>
-        Logger.error(s"[DesConnector] [getResidencyStatus] Request has timed out. userId ($userId).")
+        Logger.error(s"[DesConnector][getResidencyStatus] Request has timed out. userId ($userId).")
         Right(ResidencyStatusFailure(error_DoNotReProcess, "Internal server error."))
       case _5xx: Upstream5xxResponse =>
         if (_5xx.upstreamResponseCode == 503) {
-          Logger.error(s"[DesConnector] [getResidencyStatus] Service unavailable. userId ($userId).")
+          Logger.error(s"[DesConnector][getResidencyStatus] Service unavailable. userId ($userId).")
           Right(ResidencyStatusFailure(error_ServiceUnavailable, "Service unavailable"))
         } else {
-          Logger.error(s"[DesConnector] [getResidencyStatus] ${_5xx.upstreamResponseCode} exception caught. userId ($userId).")
+          Logger.error(s"[DesConnector][getResidencyStatus] ${_5xx.upstreamResponseCode} was returned from DES. userId ($userId).")
           Right(ResidencyStatusFailure(error_InternalServerError, "Internal server error."))
         }
       case th: Throwable =>
-        Logger.error(s"[DesConnector] [getResidencyStatus] Caught error occurred when calling the HoD. userId ($userId).Exception message: ${th.getMessage}.")
+        Logger.error(s"[DesConnector][getResidencyStatus] Caught error occurred when calling the HoD. userId ($userId).Exception message: ${th.getMessage}.")
         Right(ResidencyStatusFailure(error_InternalServerError, "Internal server error."))
       case _ =>
-        Logger.error(s"[DesConnector] [getResidencyStatus] Uncaught error occurred when calling the HoD. userId ($userId).")
+        Logger.error(s"[DesConnector][getResidencyStatus] Uncaught error occurred when calling the HoD. userId ($userId).")
```

Comment:
I think it'd be valuable to log out the exception itself in these cases. Especially the last one which is just an _ case.

Reply:
on it

Reply:
looking further into it, this is the same type of match as the one above, so I have removed this line.